communication/http/get-http-query-uri.yml

```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: get HTTP query URI
+    namespace: communication/http
+    authors:
+      - william.ballenthin@mandiant.com
+    scope: basic block
+    mbc:
+      - Communication::HTTP Communication [C0002]
+    examples:
+      - Practical Malware Analysis Lab 03-02.dll_:0x10004452
+  features:
+    - and:
+      - api: wininet.HttpQueryInfo
+      - number: 0x13 = HTTP_QUERY_URI
```

Comment:
Should this be 0xD (decimal 13) via https://learn.microsoft.com/en-us/windows/win32/wininet/query-info-flags?

Reply:
yes, definitely.

1) 🤦🏼
2) should have run with `lint --thorough` locally, but forgot
3) why didn't CI catch this with `--thorough`?

Reply:
```suggestion
      - number: 0xD = HTTP_QUERY_URI
```

Reply:
ah i need `LINT_THOROUGH` in the PR text.